src/main/resources/assets/js/app/view/PublicKeysGrid.ts

```diff
@@ -0,0 +1,124 @@
+import {DivEl} from '@enonic/lib-admin-ui/dom/DivEl';
+import {LabelEl} from '@enonic/lib-admin-ui/dom/LabelEl';
+import {AEl} from '@enonic/lib-admin-ui/dom/AEl';
+import {i18n} from '@enonic/lib-admin-ui/util/Messages';
+import {ConfirmationDialog} from '@enonic/lib-admin-ui/ui/dialog/ConfirmationDialog';
+import {PublicKey} from '../browse/serviceaccount/PublicKey';
+import {User} from '../principal/User';
+import {DeletePublicKeyRequest} from '../../graphql/principal/user/DeletePublicKeyRequest';
+import {UserKeyDetailsDialog} from '../wizard/UserKeyDetailsDialog';
+import {DateHelper} from '@enonic/lib-admin-ui/util/DateHelper';
+
+export class PublicKeysGrid
+    extends DivEl {
+
+    private readonly tbody: DivEl;
+
+    private removeCallback: () => void;
+
+    constructor(className ?: string) {
+        super(className || '');
+
+        this.getEl().addClass('public-keys-grid');
+
+        const publicKeysHeader = new LabelEl(i18n('field.userKeys.grid.title'));
+        publicKeysHeader.addClass('public-keys-label');
+        this.appendChild(publicKeysHeader);
+
+
+        const header = this.createDivEl('public-keys-row', 'rowgroup');
```

Comment:
Use a different style for the header element, to be able to apply different styling to the header row.